response.go

```diff
@@ -22,15 +24,16 @@ type Response struct {
 	hijacked bool
 }
 
-// Encode serialises the passed object as JSON into the body (and sets appropriate headers).
+// Encode serialises the passed object into the body (and sets appropriate headers).
 func (r *Response) Encode(v interface{}) {
 	if r.Response == nil {
 		r.Response = newHTTPResponse(Request{})
 	}
 
+	// If we have a proto message of a
```

Comment:
unsolicited comment: is this a sentence fragment?

Reply:
Good spot, removing 👍